src/Environment/MemoryInterface.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
+  Neighborhoods\Kojo\Environment\MemoryInterface.EnvironmentMaximumMemoryInputValue: '%env(ENVIRONMENT_MAXIMUM_MEMORY_INPUT_VALUE)%'
```

Comment:
Allows missing values since this will be a 4.x release.
Fatal error: Uncaught Symfony\Component\DependencyInjection\Exception\EnvNotFoundException: Environment variable not found: "ENVIRONMENT_MAXIMUM_MEMORY_INPUT_VALUE". in /var/www/html/kojo_fitness.neighborhoods.com/UseCase46/vendor/symfony/dependency-injection/EnvVarProcessor.php on line 131